lib/feedjira/parser/podcast.rb

```diff
@@ -19,6 +19,7 @@ class Podcast
       element :description
       element :language
       element :copyright
+      element :author
```

Comment:
nope, but so long as it sets owner:email after your fix, we'll have that correctly